internal/uidriver/glfw/input.go

```diff
@@ -243,19 +244,18 @@ func (i *Input) IsKeyPressed(key driver.Key) bool {
 	if !i.ui.isRunning() {
 		return false
 	}
-	var r bool
-	_ = i.ui.t.Call(func() error {
+	return i.ui.t.BoolCall2(uintptr(unsafe.Pointer(i)), uintptr(key), func(param1, param2 uintptr) bool {
```

Comment:
I feel like introducing a sync.Mutex would be better than adding variations of Call.